biocrnpyler/crnlab.py

```diff
@@ -35,23 +36,27 @@ def mixture(self, name, **kwargs):
             raise ValueError('Either set initial volume or the final volume')
         
 
-        filename = name + str('.csv')
-        with open(filename, 'r') as f:
-            reader = csv.reader(f)
-            # Read off the parameters
-            params = {}
-            for row in reader:
-                temp = row[1].replace('.','',1).replace('e','',1).replace('-','',1)
-                if temp.isdigit():
-                    params[str(row[0])] = float(row[1])
- 
-        if extract:
-            self.extract(extract, parameters = extract_parameters, volume = extract_volume, **kwargs)
-        if buffer:
-            self.buffer(extract, parameters = buffer_parameters, volume = buffer_volume, **kwargs)
+        try:
+            filename = name + str('.csv')
+            with open(filename, 'r') as f:
+                reader = csv.reader(f)
+                # Read off the parameters
+                params = {}
+                for row in reader:
+                    temp = row[1].replace('.','',1).replace('e','',1).replace('-','',1)
+                    if temp.isdigit():
+                        params[str(row[0])] = float(row[1])
+        except:
```

Comment:
It's a minor thing, but it'd better to specify what exception will be catched. Now, it catches everything.

Reply:
I think that's the outdated file (it says so on Github, and those lines were deleted with later commits), not sure how or why they are shown now.

Reply:
I had parameter file reading earlier when Parameter class was not ready but then I removed all that to integrate with the Parameter class.